set_environment.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 
 export LICENSE_ID=
-export SPARCDIR=${VSC_DATA}/cryoSPARC
+export SPARCDIR=${VSC_DATA}/apps/${VSC_INSTITUTE_CLUSTER}/cryoSPARC
```

Comment:
1. Using `$VSC_DATA` for software installations is probably not the best idea on Hortense, since any access has to go over the network between KU Leuven and UGent. It is probably better to use the project directory of a Tier-1 project (see https://docs.vscentrum.be/gent/tier1_hortense.html#home-on-scratch-setup)
2. cryoSPARC is essentially a self-contained conda environment, and probably contains quite a lot of files. Our colleagues from UGent would probably recommend a container installation. This might be possible, but you need to be careful about where the database is located. Maybe in first instance it is better to forget the container, but do a regular installation in `$VSC_SCRATCH_PROJECTS_BASE/your_project_name`

Reply:
sure. fixed in b70071b.